app/src/main/java/travelbook/android/app/data/TravelBookDatabaseOpenHelper.java

```diff
@@ -0,0 +1,79 @@
+package travelbook.android.app.data;
+
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+
+import java.security.PrivateKey;
+
+/**
+ * A javadoc for working with the database.
+ */
+final class MemoriesContract {
+    private MemoriesContract() {
+    }
+
+    public static class MemoriesEntry {
+        public static final String TABLE_NAME = "Memories";
+        public static final String COLUMN_NAME_MEMORY_ID = "memory_id";
+        public static final String COLUMN_NAME_MEMORY_DESC = "memory_desc";
+    }
+
+    static final String SQL_CREATE_ENTRIES =
+            "CREATE TABLE " + MemoriesEntry.TABLE_NAME + " (" +
+                    MemoriesEntry.COLUMN_NAME_MEMORY_ID + " INTEGER PRIMARY KEY," +
+                    MemoriesEntry.COLUMN_NAME_MEMORY_DESC + " TEXT)";
+}
+
+final class PhotosContract {
+    private PhotosContract() {
+    }
+
+    public static class PhotosEntry {
+        public static final String TABLE_NAME = "Photos";
+        public static final String COLUMN_NAME_PHOTO_ID = "photo_id";
+        public static final String COLUMN_NAME_PHOTO_URI = "photo_uri";
+        public static final String COLUMN_NAME_PHOTO_DESC = "photo_desc";
+        public static final String COLUMN_NAME_IS_PRIMARY = "is_primary";
+        public static final String COLUMN_NAME_MEMORY = "memory";
+        public static final String UNIQUE_INDEX = "i1";
```

Comment:
UNIQUE_INDEX is too generic and meaningless as well as value "i1".
Think of better naming.